src/newWizard/WizardStep.tsx

```diff
@@ -52,6 +53,7 @@ export default class WizardStep extends React.PureComponent<WizardStepProps> {
                 aria-hidden={id !== currentStepID}
                 className={classNameList}
             >
+                {isLoading && <Spinner size={SpinnerSize.MEDIUM} />}
```

Comment:
Just thinking, do we need the spinnerSize to be a Prop instead ?

Reply:
done